feature/experimental/isis/otg_tests/isis_hello_interval_and_multiplier_configuration_test/README.md

```diff
@@ -0,0 +1,50 @@
+# RT-2.16: IS-IS hello-interval and multiplier Configuration Test
+
+## Summary
+
+IS-IS hello-interval and multiplier Configuration Test
+
+## Procedure
+
+* Topology: ATE-port1<—> DUT–port1
+    * Configure IS-IS for ATE port-1 and DUT port-1. 
+    * Establish basic adjacency
+* Baseline Configuration:
+    * Set the hello-interval to a standard value (e.g., 10 seconds).
+    * Set the hello-multiplier to its default (typically 3) 
+    * Check that the streaming telemetry values are reported correctly.
+* Adjusting Hello-Interval:
+    * Change the hello-interval to a different value.
+    * Verify that IS-IS adjacency is coming up.
+    * Verify that the updated Hello-Interval time is reflected in isis adjacency output. 
```

Comment:
Please specify if this step is performed on DUT, ATE, or both.

Is the purpose to check the local configuration/state, or a received value from the neighbor?

Note that hello-interval is not signaled to neighbors in IS-IS. Hold-time (the product of hello-interval & hello-multiplier) is signaled, however in OpenConfig adjacency model it is not exposed.

Hence, on the DUT the only paths you can validate are IS-IS interface-level config/state values.